tests/js/articles.frontend.spec.js

```diff
@@ -0,0 +1,139 @@
+import {
+  createNewPost,
+  enablePageDialogAccept,
+  setPostContent,
+  publishPost,
+} from '@wordpress/e2e-test-utils';
+
+import {
+  selectBlockByName,
+  openSidebarPanelWithTitle,
+  clickElementByText,
+  typeInDropdownWithLabel,
+} from './e2e-tests-helpers';
+
+import 'expect-puppeteer';
+
+const ARTICLS_BLOCK = `
```

Comment:
Maybe we can make the snapshot into a template string where we call json encode? 
```
const attributes = {article_count: 2};
const ARTICLS_BLOCK = <!-- wp:planet4-blocks/articles ${JSON.stringify(attributes)} -->
<div class="wp-block-planet4-blocks-articles" data-render="planet4-blocks/articles" data-attributes="${JSON.stringify(attributes)}"></div>
<!-- /wp:planet4-blocks/articles -->
```